src/Validator/ProgrammeDateTimeNotInPast.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Validator;
+
+use Symfony\Component\Validator\Constraint;
+
+/**
+ * @Annotation
+ */
+class ProgrammeDateTimeNotInPast extends Constraint
```

Comment:
this is such a specific constraint. if you later add some date times on other entities and need a validation for a certain date time to be in the past, you cannot reuse this constrain/validator because it's for programmes. I would go for a generic constraint `DateTimeInFuture` (also because not in past means in future :smile: ) and the message should be just `This value should be a date in the future`

Reply:
So correct...